### ISSUE TITLE
Doesn't work after update!

### ISSUE THREAD

**Issue by DeadLyBro** [gildas-lormeau/SingleFile#1247]
**Describe the bug**
Extension is automatically updated and I can no longer save any web page now.

**To Reproduce**
1. Go to 'any website'
2. Click on 'extension' for save web page
3. See error.

**Expected behavior**
Save the web page.

**Screenshots**
![Screenshot_1](https://github.com/gildas-lormeau/SingleFile/assets/31214077/7c8cf078-de66-4d14-af03-6aa2187b3cc5)

**Environment**
 <html>
<body>

OS | Windows 10 Version 22H2 (Build 19045.3448)
-- | --

</body>
</html>
<html>
<body>

Yandex Browser | 116.0.5845.2271 (version 64 bit)
-- | --

</body>
</html>

**Comment by gildas-lormeau** [comment]
Did you change any setting in the options page?

**Comment by DeadLyBro** [comment]
Oh I reset all settings, now working. I think it's because you added new save option, so it got bugged. All users need to reset, or add new profile.

**Comment by gildas-lormeau** [comment]
I think I was able to identify the cause of the regression, it's due to the fact that you allowed to save scripts. This is not the default settings though. I will publish an update asap.

**Comment by gildas-lormeau** [comment]
You're welcome! Thanks also for reporting the bug :)

**Comment by DeadLyBro** [comment]
You're right! I unchecked codes (or scripts) what is `*/*` thing, I see error again, so check this if you have same error! 

![Screenshot_2](https://github.com/gildas-lormeau/SingleFile/assets/31214077/c28cc83f-d64d-4387-b302-3670d80ffbc9)

**Comment by gildas-lormeau** [comment]
I confirm I was able to reproduce and fix it, I've just uploaded the new version. Hopefully, the update should be available for download in the coming hours.

**Comment by DeadLyBro** [comment]
Thanks for help!

**Comment by DeadLyBro** [comment]
It hasn't been updated yet, right? @gildas-lormeau

**Comment by gildas-lormeau** [comment]
The extension is still in pending review, most of the time the update is published in lest than 24 hours.
